data/json/items/toolmod.json

```
@@ -16,7 +16,7 @@
     "type": "TOOLMOD",
     "category": "spare_parts",
     "name": "UPS conversion mod",
-    "description": "This device replaces conventional battery connections with a rechargeable power cell that runs off of a Unified Power System.  Having the UPS on or placing a tool in a special recharging station will slowly charge it.",
+    "description": "This device replaces conventional battery connections with a connector port that can plug into a UPS.  Any device modified with this can be used without a battery as long as you have a charged UPS with you",
```

Missed a period on end.
```suggestion
    "description": "This device replaces conventional battery connections with a connector port that can plug into a UPS.  Any device modified with this can be used without a battery as long as you have a charged UPS with you.",
```